src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './SearchBar.scss';
+
+export default class SearchBar extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { value: '' };
```

Comment:
it seems you tried to manage search value in multiple places

Reply:
The difference was between value to submit and value to type. Films filtering is not triggered by any change in the input field. So value here is just for current typing it's not necessary to be submitted therefore was not stored at SearchPage level.
But anyways, i've changed and now value is managed only by SearchPage.